plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -72,6 +72,13 @@
           - Define the state of a container.
         required: false
         default: started
+    target:
+        description:
+          - For cluster deployments. Will attempt to create a container on a target node.
+            If container exists elsewhere in a cluster, then container will not be replaced or moved.
+            The name should respond to same name of the node you see in C(lxc cluster list).
+        required: false
```

Comment:
```suggestion
        type: str
        required: false
```

Reply:
Thank you, I thought about adding type for this parameter, but looking at other parameters thought it is only required for "requited: true" options.

Reply:
Yw, so we still need an example in EXAMPLES section, users will be happy having it:)